index.html

```diff
@@ -4,9 +4,10 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
     <title>Hello Microverse Project</title>
 </head>
 <body>
-    <h1>Hello Microverse</h1>
+    <h1>Hello</h1>
 </body>
```

Comment:
- [ ] While trying to implement the required changes from the previous reviewer, you mistakenly remove `Microverse` from your `Hello Microverse`. Kindly put it back.